src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -141,6 +146,9 @@ m_cursorGrabbed   (false)
 
     if (m_handle)
     {
+        if (handleCount++ == 0)
```

Comment:
Me too. One extra line of code vs more headache when reading the code, my choice is made.